030-Concepts/050-schema.mdx

```diff
@@ -79,6 +86,7 @@ These fields are **optional**:
 - `unique`: A boolean value answering the question "is every value in this column expected to be unique?"
 - `notNull`: A boolean value answering the question "is this column exected to have data?"
 - `defaultValue`: The default value of the column, if nothing is provided.
+- `file`: Specific to [file attachments](/docs/concepts/file-attachments), the `file` field will allow you to provide the `defaultPublicAccess` of files added to that record.
```

Comment:
We might need to also document `file[]`. Which right now supports the same options, but it's for arrays

Reply:
Is it the same syntax? Like so?

```json
        {
          "name": "images",
          "type": "file[]",
          "file": {
            "defaultPublicAccess": true
          }
        }
```

Reply:
No